Landing-page.css

```diff
@@ -575,43 +540,43 @@ SVG Icons - svgicons.sparkk.fr
 
 ********************/
 
-
-
 /** 
 * COLORS
 **/
 
 .bg-black {
-    background: #2F2E41;
+  background: #2f2e41;
 }
 
 .bg-blue {
-    background: #1F59BB !important;
+  background: #1f59bb !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5575170)